### ISSUE TITLE
Feature Request: Detect duplicate variables in a module

### ISSUE THREAD

**Issue by mhmdio** [wata727/tflint#262]
```HCL
module "name" {
  source = "source"
  var1                             = "${var.var1}"
  var1                             = "${var.var1}"
}
```
should detect var1 duplication.

**Comment by mhmdio** [comment]
```HCL

**Comment by wata727** [comment]
Fortunately, such duplication will be an error in Terraform v0.12. Since TFLint 0.8.0 also strongly depends on Terraform v0.12, the following error will be reported.

```
$ tflint
Error: Failed to load configurations: template.tf:9,3-6: Attribute redefined; The argument "vpc" was already set at template.tf:8,3-6. Each argument may be set only once.
```

```tf
provider "aws" {
  region = "us-east-1"
}

module "ecs_on_spotfleet" {
  source = "github.com/wata727/tf_aws_ecs_on_spotfleet"

  vpc         = "a"
  vpc         = "a"
  subnets     = ["a", "b"]
  spot_prices = ["a", "b"]
  key_name    = "a"
}

output "endpoint" {
  value = "${module.ecs_on_spotfleet.endpoint}"
}
```
